graylog2-server/pom.xml

```diff
@@ -43,6 +43,7 @@
 
     <properties>
         <mainClass>org.graylog2.bootstrap.Main</mainClass>
+        <webInterface.path>../graylog2-web-interface</webInterface.path>
```

Comment:
Ideally I'd like to see the entire web-interface related code to live in a separate module, not being tied into the server build process.
But I guess we can refactor this afterwards in the interest of limiting the branch lifetime.